WebService/v1/Models/SimulationApiModel/SimulationIotHub.cs

```diff
@@ -14,15 +14,28 @@ public class SimulationIotHub
         [JsonProperty(PropertyName = "ConnectionString")]
         public string ConnectionString { get; set; }
 
+        [JsonProperty(PropertyName = "PreprovisionedIoTHubInUse")]
+        public bool PreprovisionedIoTHubInUse { get; set; }
+
+        [JsonProperty(PropertyName = "PreprovisionedIoTHubMetricsUrl")]
+        public string PreprovisionedIoTHubMetricsUrl { get; set; }
```

Comment:
note: throughout the code we have ["hub preprovisioned" "preprovisioned hub"] and ["preprovisioned iot hub"]. It would be nice to align to one of the two, i.e. with or without the "iot" word

Reply:
We will address this is a separate PR. Already scope of this PR has increased a lot

Reply:
:white_check_mark: